exercises/concept/calculator-conundrum/.meta/src/reference/java/CalculatorConundrum.java

```diff
@@ -0,0 +1,33 @@
+public class CalculatorConundrum {
+
+    public String calculate(int operand1, int operand2, String operation) {
+        int result;
+        try {
+            if (operation == null) {
+                throw new IllegalArgumentException("Operation cannot be null");
+            }
+            switch (operation) {
+                case "+":
+                    result = SimpleOperation.addition(operand1, operand2);
+                    break;
+                case "*":
+                    result = SimpleOperation.multiplication(operand1, operand2);
+                    break;
+                case "/":
+                    result = SimpleOperation.division(operand1, operand2);
```

Comment:
I would put the try-catch for the ArithmeticException right around this line to clearly scope where that exception might be thrown.